views/default/object/elements/full/body.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Outputs object full view
+ * @uses $vars['body'] Body
+ */
+
+$body = elgg_extract('body', $vars);
+if (!$body) {
+	return;
+}
+?>
+<div class="elgg-listing-full-body"><?= $body ?></div>
```

Comment:
This changes the markup, but I'd rather change it than have:
`html
<div class="elgg-content">
   <div class="elgg-content"></div>
   <div class="elgg-output"></div>
</div>
`
Makes it impossible to target the view.